tracks/elixir/exercises/roman-numerals/README.md

```diff
@@ -0,0 +1,79 @@
+### Reasonable solutions
+
+```elixir
+defmodule Roman do
+  @dict [
+    {1000, "M"},
+    {900, "CM"},
+    {500, "D"},
+    {400, "CD"},
+    {100, "C"},
+    {90, "XC"},
+    {50, "L"},
+    {40, "XL"},
+    {10, "X"},
+    {9, "IX"},
+    {5, "V"},
+    {4, "IV"},
+    {1, "I"}
+  ]
+
+  def numerals(0, acc), do: acc
+
+  def numerals(number, acc) do
+    {arabic, roman} = @dict |> Enum.find(fn {n, _} -> n <= number end)
```

Comment:
Love this approach, I've seen similar approaches and felt it could be made even simpler/less functions/writing. Glad I could learn from your approach :D 💜